.buildkite/commands/prototype-build-jetpack.sh

```diff
@@ -1,7 +1,6 @@
 #!/bin/bash -eu
 
-# FIXIT-13.1: Prototype Builds want the latest version of Sentry CLI
```

Comment:
Random question: does this "FIXIT-13.1" comment mean something like "this issue will be fixed in 13.1 version of the VM image"?

Reply:
I think so... It comes from [a commit](https://github.com/wordpress-mobile/WordPress-iOS/commit/a5a518010ee74b4e81cdd839c3a543fbe8b0fa3f) that explicitly mentions having issues in Xcode 13.